internal/proxy/proxy.go

```diff
@@ -310,8 +310,16 @@ func (node *Proxy) sendChannelsTimeTickLoop() {
 					Timestamps:       tss,
 					DefaultTimestamp: maxTs,
 				}
-				sub := tsoutil.SubByNow(maxTs)
-				metrics.ProxySyncTimeTickLag.WithLabelValues(strconv.FormatInt(paramtable.GetNodeID(), 10), "default").Set(float64(sub))
+
+				func() {
+					for channel, ts := range stats {
+						sub := tsoutil.SubByNow(ts)
```

Comment:
using maxTs should be enough, generally, we pay more attention to max lag.

Reply:
Normally, the default ts won't lag the latest time too much, so the metric turns to be useless.

Reply:
Btw, we should use the min ts to report the lag if we want to pay more attention to max lag.